js/components/bookmarksToolbar.js

```diff
@@ -215,19 +215,18 @@ class BookmarksToolbar extends ImmutableComponent {
       if (a && a.href) {
         appActions.addSite({
           title: a.innerText,
-          location: a.href
+          location: e.dataTransfer.getData('text/plain')
```

Comment:
a.href was truncated sometimes and I'm not sure why, but text/plain always contained the correct link text